src/modules/SideBarDrawer/HelpDrawer.js

```diff
@@ -25,7 +23,11 @@ const HelpDrawer = ({helpText, skillInfo, skillsOnly, onClose, classes}) => {
   };
   return (
     <Paper className={classes.BasicInfoPaper}>
-      <Grid container direction="column">
+      <Grid
+        container
+        direction="column"
+        style={openDrawer2 ? {display: 'none'} : null}
```

Comment:
The name "openDrawer2" is kind of vague (I originally thought there was another variable named "openDrawer" and we needed two variables for some reason). Maybe we could change the name to be more explicit, like "openSecondDrawer"?

Reply:
I will fix it later